utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -237,6 +273,46 @@ inline void set_descriptor_layout( RpptDescPtr srcDescPtr, RpptDescPtr dstDescPt
     }
 }
 
+// sets roi xywh values and dstImg sizes
+inline void set_roi(vector<string>::const_iterator imagePathsStart, vector<string>::const_iterator imagePathsEnd, RpptROI *roiTensorPtrSrc, RpptROI *roiTensorPtrDst, RpptImagePatchPtr dstImgSizes)
```

Comment:
Fix the alignment with dstImgSizes parameter

Reply:
Aligment is proper, Not required